src/wallet/isWalletASmartWallet.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  CB_SW_PROXY_BYTECODE,
+  CB_SW_V1_IMPLEMENTATION_ADDRESS,
+  ERC_1967_PROXY_IMPLEMENTATION_SLOT,
+} from './constants';
+import { IsWalletASmartWalletOptions } from './types';
+import { decodeAbiParameters } from 'viem';
+import type { Address, BlockTag, Hex } from 'viem';
```

Comment:
overall all of these import should be
```
import { decodeAbiParameters } from 'viem';
import {
  CB_SW_PROXY_BYTECODE,
  CB_SW_V1_IMPLEMENTATION_ADDRESS,
  ERC_1967_PROXY_IMPLEMENTATION_SLOT,
} from './constants';
import type { Address, BlockTag, Hex } from 'viem';
import type { IsWalletASmartWalletOptions } from './types';
```

Reply:
I'm still confused about how we ordered these.

Reply:
I suppose that packages are always before files

Reply:
Yup

it's
- packages
- files
- types